src/pages/settings/Payments/PaymentsPage.js

```diff
@@ -216,6 +216,9 @@ class PaymentsPage extends React.Component {
 
     render() {
         const isPayPalMeSelected = this.state.formattedSelectedPaymentMethod.type === CONST.PAYMENT_METHODS.PAYPAL;
+
+        // determine if the popover appears from the bottom of the screen or not.
```

Comment:
Looking for a more detailed comment, something along the lines of
`Determines whether or not the modal popup is mounted from the bottom of the screen instead of the side mount on Web or Desktop screens`

Reply:
looks better, i update it now